Geometry/MTDNumberingBuilder/interface/CmsMTDLevelBuilder.h

```diff
@@ -0,0 +1,49 @@
+#ifndef Geometry_MTDNumberingBuilder_CmsMTDLevelBuilder_H
```

Comment:
It looks like this file is used in plugins only. I don't think it should be moved in an interface directory which is a separate library. If it helps, it can be included without subsystem/package/ path:
```
#include "CmsMTDLevelBuilder.h"
```

Reply:
Thank you, I was not aware of that policy.

Reply:
The framework uses just the local name for “private” headers that reside in the src directory.